src/scenes/ApplicationRouter/portal/MapUpdate/index.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+
+import { Box, Card, Grid, Typography, useTheme } from '@mui/material';
+import { Button } from '@terraware/web-components';
+import area from '@turf/area';
+import { Feature, FeatureCollection } from 'geojson';
+
+import { Crumb } from 'src/components/BreadCrumbs';
+import EditableMap from 'src/components/Map/EditableMapV2';
+import { unionMultiPolygons } from 'src/components/Map/utils';
+import { APP_PATHS } from 'src/constants';
+import useNavigateTo from 'src/hooks/useNavigateTo';
+import useUndoRedoState from 'src/hooks/useUndoRedoState';
+import { useLocalization } from 'src/providers';
+import { requestUpdateApplicationBoundary } from 'src/redux/features/application/applicationAsyncThunks';
+import { selectApplicationUpdateBoundary } from 'src/redux/features/application/applicationSelectors';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import { SQ_M_TO_HECTARES } from 'src/scenes/PlantingSitesRouter/edit/editor/utils';
+import strings from 'src/strings';
+import { MultiPolygon } from 'src/types/Tracking';
+import useSnackbar from 'src/utils/useSnackbar';
+
+import { useApplicationData } from '../../provider/Context';
+import ApplicationPage from '../ApplicationPage';
+
+// undo redo stack to capture site boundary and errors
+type Stack = {
+  errorAnnotations?: Feature[];
+  siteBoundary?: FeatureCollection;
+};
+
+const MapView = () => {
+  const theme = useTheme();
+
+  const { activeLocale } = useLocalization();
+  const dispatch = useAppDispatch();
+  const { selectedApplication, reload } = useApplicationData();
+  const { goToApplicationMap } = useNavigateTo();
+  const { toastSuccess } = useSnackbar();
+  const [siteBoundaryData, setSiteBoundaryData, undo, redo] = useUndoRedoState<Stack>();
+
+  const [requestId, setRequestId] = useState<string>('');
+  const result = useAppSelector(selectApplicationUpdateBoundary(requestId));
+
+  const findErrors = (boundary: MultiPolygon) => {
+    const boundaryAreaHa = parseFloat((area(boundary) * SQ_M_TO_HECTARES).toFixed(2));
+    const maxAreaHa = 20000;
```

Comment:
Can you make this a top level const similar to `SQ_M_TO_HECTARES`? It can be towards the top of this file or the same file as that other const.